lib/rspec/rails/fixture_support.rb

```diff
@@ -9,11 +9,11 @@ module FixtureSupport
         include RSpec::Rails::MinitestAssertionAdapter
         include ActiveRecord::TestFixtures
 
-        if ::Rails.version.to_f >= 6.1
-          # @private return the example name for TestFixtures
-          def name
-            @example
-          end
+        # @private prevent ActiveSupport::TestFixtures to start a DB transaction.
+        # Monkey patched to avoid collisions with 'let(:name)' in Rails 6.1 and after
+        # and let(:method_name) before Rails 6.1.
+        def run_in_transaction?
+          use_transactional_tests && !self.class.uses_transaction?(@example)
```

Comment:
Not “reviewing,” just curious - were you able to determine whether any RSpec code still sets `@example` to a meaningful value?

Reply:
No I didn't search this yet. But I will.

Reply:
For us this probably needs to be `example` or `self` as we don't assign `@example` within a test like mini test does afaik